app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt

```diff
@@ -59,6 +61,8 @@ class DetailsFragment : BottomSheetDialogFragment() {
             binding.end.text = "${setTimeString(args.ehr)}:${setTimeString(args.emin)}"
             if (args.vibSwitch) binding.audioMode.setImageResource(R.drawable.vibration)
             else binding.audioMode.setImageResource(R.drawable.mute)
+            if (args.repeatWeekly)binding.repeatWeeklyIcon.visibility = VISIBLE
```

Comment:
change to binding.repeatWeeklyIcon.visibility = if (args.repeatWeekly) ....

Reply:
@rob729 Done